config.py

```diff
@@ -1,7 +1,9 @@
-user = "root"
-password = "root"
+user = "hritik"
+password = "****"
 host = "localhost"
 database = "mapbot"
+stanford_path_to_jar="./stanford-corenlp-3.9.2.jar"
+stanford_paht_to_models_jar="./stanford-corenlp-3.9.22-models.jar"
 #key = "*Google_Cloud_API_key*" #Will be provided by mentors
 #stanford_path_to_jar = "*your_path_to_stanford-corenlp-x.x.x.jar*" #for eg. "stanford-corenlp-full-2018-10-05/stanford-corenlp-3.9.2.jar"
 #stanford_path_to_models_jar = "*your_path_to_stanford-corenlp-x.x.x-models.jar*" #for eg. "stanford-corenlp-full-2018-10-05/stanford-corenlp-3.9.2-models.jar"
```

Comment:
`config.py` should be in .gitignore as explained in the initial setup. This file should never be pushed.